src/tensorflow_backend_tf.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+// Copyright 2019-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Just confirming, you copied the file from TensorFlow repository directly here, right?

Reply:
Correct.